src/settings-ui/Settings.UI/FlyoutWindow.xaml

```diff
@@ -26,7 +26,6 @@
             LightTintColor="#F3F3F3"
             LightTintOpacity="0" />
     </winuiex:WindowEx.Backdrop>
-    <Grid>
-        <flyout:ShellPage x:Name="FlyoutShellPage"/>
-    </Grid>
+    <flyout:ShellPage x:Name="FlyoutShellPage"
+                      Background="{ThemeResource AcrylicBackgroundFillColorBaseBrush}" />
```

Comment:
This appears to be overriding the acrylic effect. Without the `Background` being set here, the acrylic works as expected transparency wise. Unfortunately, this uses the WinUIEx acrylic theme which does not add the noise layer like the regular acrylic.

I couldn't get the noise to show on the window, or get the brush to be transparent. Not sure what is going on here.

Thoughts?

Reply:
@niels9001